CSS/index.css

```diff
@@ -106,4 +111,103 @@ input#menuButton {
     transform: translateY(-305px);
     transition: transform 500ms;
   }
+
+  #bGImage {
+    margin-top: -213px;
```

Comment:
Not really Im not sure why youre doing this. Shouldnt need the margin I dont think.

Reply:
comment it out, and shrink the page from desktop to mobile. If you have another way to fix that I'm game.

Reply:
Should be able to fix this issue without it I think.